### ISSUE TITLE
default keyboard shortcuts

### ISSUE THREAD

**Issue by tabetomo** [pgourlain/vscode_erlang#1]
I found shift+cmd+b does't work as intended, i.e. always says **"starting rebar compile ... "**. when I codes python, ruby, and nodejs, etc.  It seems the extension overwrites the "default keyboard shortcuts",
{ "key": "shift+cmd+b",           "command": "workbench.action.tasks.build" },
with the erlang specific shortcut,
{ "key": "shift+cmd+b",           "command": "extension.rebarBuild" },

The problem is that I cannot disable the shortcuts since there seems no config files for the default keyboard shortcuts in vscode. (vscode seems register the key binding into a registry by a code
`workbenchActionsRegistry.registerWorkbenchAction(new SyncActionDescriptor(BuildAction, BuildAction.ID, BuildAction.TEXT, { primary: KeyMod.CtrlCmd | KeyMod.Shift | KeyCode.KEY_B }), 'Tasks: Run Build Task', tasksCategory);` in vscode/src/vs/workbench/parts/tasks/electron-browser/task.contribution.ts and I don't know how to change the registry ).

So I have to remove the extension!!! when writing other language (if I remove the extension, the shortcuts also vanish).

The solution can be easy. it is better to include **"when"** like (this is rust's example)

```
{ "key": "ctrl+shift+t",          "command": "rust.cargo.test.debug",
                                     "when": "editorLangId == 'rust'" }, 
```

in vscode_erlang/package.json

```
        "keybindings": [
            { "command": "extension.rebarBuild", "mac": "shift+cmd+b", "key": "ctrl+shift+b"},
            { "command": "extension.erleunit", "mac": "shift+cmd+t", "key": "ctrl+shift+t"}
        ],
```

**Comment by pgourlain** [comment]
Thanks for your report !
i'll fix it.

**Comment by pgourlain** [comment]
fix in 0.9
